README.md

```diff
@@ -275,4 +275,11 @@ location / {
 	# backend
 	proxy_pass http://127.0.0.1:8000;
 }
-```
\ No newline at end of file
+```
+#### Setting up SMTP server
+The current code works with SSL-based SMTP server using username and password as authentication. Below are the points to note for particular SMTP server:
+
+##### Google SMTP
```

Comment:
For this section, can you write these instructions instead:

```
1. Turn on the "Allow less secure apps" setting in your Google account: https://support.google.com/accounts/answer/6010255

2. If you use 2-Factor Authentication for your account, you will have to generate an app password: https://support.google.com/accounts/answer/185833

3. Use these settings in `politeiawww.conf`:

       mailhost=smtp.gmail.com:465
       mailuser=<your email address>
       mailpass=<your password or app password>
       webserveraddress=https://localhost:3000
```

Reply:
For point 2, did you test it before?

Reply:
Yeah, I have 2FA enabled so I had to go through that process.

Reply:
Good. I will change the README.md accordingly.